enginetest/enginetests.go

```diff
@@ -1699,6 +1698,21 @@ func TestRenameColumn(t *testing.T, harness Harness) {
 		[]sql.Row{
 			{1, "first row"},
 		}, nil, nil)
+
+	// Test column rename behavior when a table check constraint is involved
+	RunQuery(t, e, harness, "ALTER TABLE mytable ADD CONSTRAINT test_check CHECK (i2 < 12345)")
+	AssertErr(t, e, harness, "ALTER TABLE mytable RENAME COLUMN i2 TO i3", sql.ErrCheckConstraintInvalidatedByColumnAlter)
+	RunQuery(t, e, harness, "ALTER TABLE mytable RENAME COLUMN s2 TO s3")
+	tbl, ok, err = db.GetTableInsensitive(NewContext(harness), "mytable")
```

Comment:
Use empty lines to break up these assertions into logical groupings

Reply:
✅